livekit/room.py

```diff
@@ -34,6 +36,7 @@
 class RoomOptions:
     auto_subscribe: bool = True
     dynacast: bool = True
+    e2ee_options: Optional[E2EEOptions] = None
```

Comment:
since they are all options, `_options` suffix seems redundant

```suggestion
    e2ee: Optional[E2EEOptions] = None
```